proposals/drafts/disclosures.rst

```diff
@@ -0,0 +1,806 @@
+
+forms
+    **optional**
+    A list of forms associated with this disclosure. See ``Form`` object below.
+
+schedules
+    **optional**
+    A list of schedules associated with this disclosure. See ``Schedule`` object below.
+
+official_identifier
+    **optional**
+    Upstream identifier of the disclosure if one exists, such as the filing ID assigned by the Senate Office of Public Record
+
+submitted_date
+    **optional**
+    Date (and possibly time) when document was submitted.
+
+effective_date
+    **optional**
+    Effective date of the registration. (May be retroactive, ie, earlier than submitted date).
+
+created_at
+    Time that this object was created at in the system, not to be confused with the date of
+    introduction.
+
+updated_at
+    Time that this object was last updated in the system, not to be confused with the last action.
+
+documents
+    All documents related to the disclosure with the exception of versions (which are part of
+    the above ``versions``).
+
+    note
+        Note describing the document's relation to the disclosure (e.g. 'submitted filing', 'request for additional information', etc.)
+    date
+        The date the document was published in YYYY-MM-DD format
+        (partial dates are acceptable).
+    links
+        Links to 'available forms' of the document.  Each document can be available in
+        multiple forms such as PDF and HTML.  (For those familiar with DCAT this is the same
+        as the ``Distribution`` class.)
+        Has the following properties:
+
+        url
+            URL of the link.
+        media_type
+            The `media type <http://en.wikipedia.org/wiki/Internet_media_type>`_ of the link.
+
+sources
+    List of sources used in assembling this object.  Has the following properties:
+
+    url
+        URL of the resource.
+    note
+        **optional**
+        Description of what this source was used for.
+
+extras
+    Common to all Open Civic Data types, the value is a key-value store suitable for storing arbitrary information not covered elsewhere.
+
+Disclosure Type
+~~~~~~~~~~~~~~~
+
+identifier
+    An identifier that uniquely identifies the disclosure type.
+
+name
+    The canonical name of the disclosure type
+
+description
+    Description of the disclosure type
+
+action
+    The action performed by this disclosure type. Current values include:
+    
+    * registration  - registers a person or organization with a DisclosureAuthority
+    * report        - makes a periodic report to a DisclosureAuthority
+
+classification
+    The category of the disclosure type. Current values include:
+        
+    * lobbying      - Disclosures related to lobbying
+    * contributions - Disclosures related to political contributions
+
+amends_type
+    The identifier of the disclosure type that this disclosure type is able to amend. Can be the same as identifier, where future submissions supercede past submissions.
```

Comment:
the classification refers more to the general area of concern that a disclosure is submitted in.  a theater or arena of disclosure, as it were. eg, lobbying diclosures can be one of "registration" or "report." the same set of actions happen to be available in the contributions world, but that needn't be the case.